src/test/java/org/broadinstitute/hellbender/tools/funcotator/vcfOutput/VcfOutputRendererUnitTest.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.hellbender.tools.funcotator.vcfOutput;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+
+/**
+ * Unit test class for the {@link VcfOutputRenderer} class.
+ * Created by jonn on 9/1/17.
+ */
+public class VcfOutputRendererUnitTest extends GATKBaseTest {
```

Comment:
Was this unit test ever filled in? I think that `VcfOutputRenderer` should definitely have unit tests.

Reply:
It hasn't been filled out yet.  I will fill it in when we collapse the rest of the branches.